docs/source/api.rst

```diff
@@ -18,6 +18,9 @@ Analytics
 .. autoclass:: clx.analytics.dga_detector.DGADetector
     :members:
 
+.. autoclass:: clx.eda.EDA
```

Comment:
doc build is not picking up the eda class. not sure why. i'm able to import it and run eda unit tests in my environment:
```
WARNING: autodoc: failed to import class 'eda.EDA' from module 'clx'; the following exception was raised:
Traceback (most recent call last):
  File "/opt/conda/envs/rapids/lib/python3.7/site-packages/sphinx/ext/autodoc/importer.py", line 67, in import_module
    return importlib.import_module(modname)
  File "/opt/conda/envs/rapids/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/conda/envs/rapids/lib/python3.7/site-packages/clx-0.18.0a0+35.gb2fa1f6.dirty-py3.7.egg/clx/eda/__init__.py", line 1, in <module>
    from clx.eda.eda import EDA  # noqa: F401
  File "/opt/conda/envs/rapids/lib/python3.7/site-packages/clx-0.18.0a0+35.gb2fa1f6.dirty-py3.7.egg/clx/eda/eda.py", line 18, in <module>
    import cuxfilter
  File "/opt/conda/envs/rapids/lib/python3.7/site-packages/cuxfilter/__init__.py", line 1, in <module>
    from .dataframe import DataFrame
  File "/opt/conda/envs/rapids/lib/python3.7/site-packages/cuxfilter/dataframe.py", line 5, in <module>
    from .dashboard import DashBoard
  File "/opt/conda/envs/rapids/lib/python3.7/site-packages/cuxfilter/dashboard.py", line 10, in <module>
    from .charts.core import BaseChart, BaseWidget, ViewDataFrame
  File "/opt/conda/envs/rapids/lib/python3.7/site-packages/cuxfilter/charts/__init__.py", line 7, in <module>
    from .datashader import scatter, stacked_lines, heatmap, graph
  File "/opt/conda/envs/rapids/lib/python3.7/site-packages/cuxfilter/charts/datashader/__init__.py", line 1, in <module>
    from .datashader import (
  File "/opt/conda/envs/rapids/lib/python3.7/site-packages/cuxfilter/charts/datashader/datashader.py", line 1, in <module>
    from . import plots
  File "/opt/conda/envs/rapids/lib/python3.7/site-packages/cuxfilter/charts/datashader/plots.py", line 1, in <module>
    from ..core.non_aggregate import (
  File "/opt/conda/envs/rapids/lib/python3.7/site-packages/cuxfilter/charts/core/non_aggregate/__init__.py", line 1, in <module>
    from .core_scatter import BaseScatter
  File "/opt/conda/envs/rapids/lib/python3.7/site-packages/cuxfilter/charts/core/non_aggregate/core_scatter.py", line 3, in <module>
    from .core_non_aggregate import BaseNonAggregate
  File "/opt/conda/envs/rapids/lib/python3.7/site-packages/cuxfilter/charts/core/non_aggregate/core_non_aggregate.py", line 3, in <module>
    import cuspatial
  File "/opt/conda/envs/rapids/lib/python3.7/site-packages/cuspatial/__init__.py", line 1, in <module>
    from .core import interpolate
  File "/opt/conda/envs/rapids/lib/python3.7/site-packages/cuspatial/core/interpolate.py", line 8, in <module>
    from cuspatial._lib.interpolate import (
  File "cuspatial/_lib/interpolate.pyx", line 1, in init cuspatial._lib.interpolate
TypeError: rmm._cuda.stream.Stream is not a type object
```

Reply:
Thanks! I'll take a look too

Reply:
I think I was able to resolve this in `docs/source/conf.py`. Let me know what you think

Reply:
looks good now.